docs/development.md

```diff
@@ -2,6 +2,22 @@
 
 As simple as possible development setup for all three major parts of OpenScanHub (OSH) pipeline. We want to make it compatible with Python 3.6 (the main Python in RHEL 8, supported for its whole lifetime) so we use that specific version inside and outside of the container.
 
+## Quick setup
+
+Execute following commands to quickly try a local OpenScanHub deployment (through podman/docker compose):
+
+```
+git clone git@github.com:openscanhub/openscanhub.git
+cd openscanhub
+git clone git@github.com:release-engineering/kobo.git
```

Comment:
Does SSH cloning work even when you don't have commit rights or any SSH keys set?